react-hot-button-component/src/hot-buttton.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+let count = 0;
```

Comment:
You don't need an external variable for this, just store `count` in state instead of `isClicked`.